config/localization/en/app.json

```diff
@@ -978,7 +979,8 @@
       "SLIPPAGE": "Slippage",
       "SUBJECT_TO_RULES": "Subject to these rules",
       "SWAP": "Swap",
-      "ESTIMATED_TIME": "Estimated Time"
+      "ESTIMATED_TIME": "Estimated Time",
+      "DEPOSIT_ACKNOWLEDGEMENT": "I agree to the {TERMS} and {POLICY}, including that I accept the risks associated with these products and services and will not use them from the United States or any other restricted jurisdiction. {VIEW_MORE_LINK}"
```

Comment:
do we also need keys for "Policy" and "Terms" (or are we using terms of use) to be added? (doesn't seem like they're in yet) // also the english of "..., including that I ..." feels mildly clunky but that's just a nit and fine if it's what legal prefers

Reply:
Oh the keys are links to Terms of use and Privacy Policy pages. The wording should have came from legal as well.

Reply:
are `TERMS` and `POLICY` going to be hyperlinks/anchor tags? 
might be more accurate/descriptive to rename them: `TERMS_LINK` and `POLICY_LINK` if so

Reply:
I can rename